src/vsphere_cpi/lib/cloud/vsphere/storage_list.rb

```diff
@@ -0,0 +1,28 @@
+module VSphereCloud
+  module StorageList
+
+    # storageList is a list of datastores and datastore_clusters defined in cloud_properties as 'datastores'
```

Comment:
Let's put this comment above immediately above the `module` since that's the style used by YARD and `rdoc`.